### ISSUE TITLE
Not in npm registry? node-bepop

### ISSUE THREAD

**Issue by M0rph3v5** [hybridgroup/node-bebop#3]

**Comment by deadprogram** [comment]
Looks like it is there to me: https://www.npmjs.com/package/node-bebop

Was it possible you were having network problems?

**Comment by M0rph3v5** [comment]
Tried, gave me an error in terminal of no such package. Then tried
searching for it on the npm website again, couldn't find anything when
searching for bebop.
Weird. Let me try again when I get home in a sec.

**Comment by deadprogram** [comment]
Hi, @M0rph3v5 

To install this module, you need to use `npm install node-bebop` as listed here: https://github.com/hybridgroup/node-bebop#how-to-install

Is that how you tried to install?

**Comment by M0rph3v5** [comment]
Lol, omg. I feel so stupid, I was typing npm install node-be*p*op ....
/facepalm

2015-06-19 23:13 GMT+02:00 Ron Evans <notifications@github.com>:
